.appveyor.yml

```diff
@@ -3,6 +3,9 @@ image: Visual Studio 2017
 cache:
     - c:\deps -> .appveyor.yml
     - c:\spellcheck-dicts -> .appveyor.yml
+    - c:\Tools\vcpkg\installed
+init:
+  - ps: iex ((new-object net.webclient).DownloadString('https://raw.githubusercontent.com/appveyor/ci/master/scripts/enable-rdp.ps1')) #rdp debug
```

Comment:
What is this used for?

Reply:
sorry, i forget to remove it, it's rdp testing. fix soon